examples/tutorials/01_basics/06_sing_scales.py

```diff
@@ -16,23 +16,21 @@
 
 '''Make Cozmo sing Do Re Mi.
 
-Slight extension from hello_world.py - introduces for loops to make Cozmo "sing" the scales.
+Slight extension from the earlier 03_count.py example.
+Make Cozmo "sing" the scales.
 '''
 
-import sys
-
 import cozmo
 from cozmo.util import degrees
 
-def run(sdk_conn):
-    '''The run method runs once Cozmo is connected.'''
-    robot = sdk_conn.wait_for_robot()
 
+def cozmo_program(robot: cozmo.robot.Robot):
+    # scales is a list of the notes for Cozmo to sing
```

Comment:
I think "notes" should be changed to "words" or similar.

Is this actually working with him changing the tones with voice_pitch, by the way?